src/main/java/com/extendedclip/papi/expansion/player/PlayerExpansion.java

```diff
@@ -447,6 +447,21 @@ public String onRequest(OfflinePlayer player, String identifier) {
                 return bool(p.isLeashed());
             case "is_inside_vehicle":
                 return bool(p.isInsideVehicle());
+            case "potion_has": {
```

Comment:
wouldn't it make more sense for this to be called something like "has_potion_effect_<effect-name>"?

Reply:
Perhaps. I just wanted to have the same prefix to be easier to search. I guess I will have to sacrifice that for more readability.

Reply:
I think that is the best. And when someone goes to the list of placeholders it would be probably better for them to be able to search directly for what they want "has potion effect" instead of having to see what the standard is and go thru each placeholder.